docs/swagger/spec.yml

```diff
@@ -3156,6 +3156,9 @@ definitions:
         description: output export path
       render:
         $ref: '#/definitions/Render'
+      dropboxCredential:
```

Comment:
Ah my fault, I didn't realize this is only generated on demand that it comes from the [user](https://github.com/azavea/raster-foundry/pull/1809/files/087d20ffff4460613bca3ede828306176fb3afd8#diff-5c2cd15f84118de8af24e9a91a6c13f8R197)